keyboards/4pplet/waffling60/rev_e/info.json

```
@@ -0,0 +1,651 @@
+{
+    "manufacturer": "4pplet",
+    "keyboard_name": "waffling60 Rev E",
+    "maintainer": "4pplet",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "A2", "pin_b": "A1", "resolution": 2}
+        ]
+    },
+    "features": {
+        "audio": false,
+        "backlight": false,
```

not enabled by default
```suggestion
        "audio": false,
        "backlight": false,
```